web_renderer/src/server.ts

```diff
@@ -0,0 +1,71 @@
+import express, { Request, Response } from 'express';
+import { SessionId } from './common';
+import { Session } from './session';
+import { constants as HttpConstants } from 'http2';
+import { randomUUID } from 'crypto';
+import { GetFrameRequest, NewSessionRequest } from './schemas';
+import { sessions } from './state';
+
+const app = express();
+
+app.use(express.json());
+
+app.post("/new_session", (req: Request, res: Response<NewSessionResponse>) => {
+    try {
+        const data = NewSessionRequest.parse(req.body);
+        console.log(`Starting rendering for ${data.url}`);
```

Comment:
it's fine for now, but keep in mind that if we stay with this approach we will need real logger